files/image_config/copp/copp_cfg.j2

```diff
@@ -1,6 +1,7 @@
 {
     "COPP_GROUP": {
 	    "default": {
+		    "trap_action":"trap",
```

Comment:
This file is not validated against yang. The trap action for default is hardcoded due to legacy code present in https://github.com/sonic-net/sonic-swss/blob/465391d1825ea3906d27a1285d2da5876f763cd7/orchagent/copporch.cpp#L185

By default the copp config doesn't go to config_db and thus we don't require yang validation on this file. However if the user need to override the copp config, user needs to set the appropriate fields along with correct yang validation.